src/app/clusters/ota-requestor/ExtendedOTARequestorDriver.cpp

```diff
@@ -35,7 +35,7 @@ bool ExtendedOTARequestorDriver::CanConsent()
                ChipLogProgress(SoftwareUpdate, "Failed to get local config disabled, using as false"));
 
     // If local config is disabled, we can't consent.
-    return localConfigDisabled == false;
+    return localConfigDisabled == true;
```

Comment:
I think the previous code was OK. Only if `localConfigDisabled == false`, the requestor can consent as described in the "User consent delegation to Nodes" section of the spec. Am I missing something?

Reply:
I missed the part talking about this in the spec and reading the comment, I thought it had implemented the opposite logic. I think I'll just update the comment to better reflect the spec.

Also, it seems given certain platforms like Darwin will always have `LocalConfigDisabled = false`, that means the `RequestorCanConset` field will always be 1, whether `--requestorCanConsent` field is supplied or not. I'll update that command line option so the field can be specified explicitly when supplied.